aptos-move/e2e-testsuite/src/tests/transaction_fuzzer.rs

```diff
@@ -70,24 +70,24 @@ proptest! {
         let num_accounts = accounts.len();
 
         for (i, txn) in txns.into_iter().enumerate() {
-            let script = txn.encode();
+            let payload = txn.encode();
             let (account, account_sequence_number) = accounts.get_mut(i % num_accounts).unwrap();
-            let script_is_rotate = LegacyStdlibScript::try_from(script.code()).map(|script|
+            /*let script_is_rotate = LegacyStdlibScript::try_from(script.code()).map(|script|
                 script == LegacyStdlibScript::RotateAuthenticationKey ||
                 script == LegacyStdlibScript::RotateAuthenticationKeyWithNonce ||
                 script == LegacyStdlibScript::RotateAuthenticationKeyWithRecoveryAddress
-            ).unwrap_or(false);
+            ).unwrap_or(false); */
```

Comment:
delete?

Reply:
This was done to pass the compile error. 
I delete this test completely now